packages/cra-template/template/README.md

```diff
@@ -0,0 +1,60 @@
+# Getting Started with Create React App and Fluent UI
```

Comment:
nit: do we need a README for both `cra-template` and `cra-template/template` ?

Reply:
those are different things

- Readme in root package is readme for the package
- Readme in template is standard CRA readme that will be added when you boot up new app via cra

Reply:
Ahh that makes sense, thanks for clarifying